include/MixerView.h

```diff
@@ -97,7 +102,17 @@ public slots:
 
 private slots:
 	void updateFaders();
+	// TODO This should be improved. Currently the solo and mute models are connected via
+	// the MixerChannelView's constructor with the MixerView. It would already be an improvement
+	// if the MixerView connected itself to each new MixerChannel that it creates/handles.
 	void toggledSolo();
+	void toggledMute();
+
+private:
+	Mixer* getMixer() const;
```

Comment:
The rationale for the naming is explained in the message of commit 37f97344d07: "It's called `getMixer` and not just `mixer` because it allows for locale variables to be called `mixer` without confusing it with the method."

I think it makes the code more readable. Otherwise you have code like:
```
auto * m = mixer();
m->doThis();
```
Or:
```
auto * mix = mixer();
mix->doThat();
```

I think the code is more comprehensible if the local variable can be called `mixer`:
```
auto * mixer = getMixer();
mixer->doFoo();
```